src/main/java/uk/gov/di/ipv/cri/common/library/domain/personidentity/Address.java

```diff
@@ -153,14 +153,23 @@ public void setValidUntil(LocalDate validUntil) {
 
     @JsonIgnore
     public AddressType getAddressType() {
-        if (Objects.nonNull(this.getValidUntil()) && isPastDate(this.getValidUntil())) {
-            return AddressType.PREVIOUS;
+
```

Comment:
declare some variables to use in the various conditional statement below to make the code a little more legible:
`boolean validFromSpecified = Objects.nonNull(this.getValidFrom());`
`boolean validUntilSpecified = Objects.nonNull(this.getValidUntil());`

Reply:
Conditionals have been used to improve legibility and if statements refactored.